src/views/Home.jsx

```diff
@@ -1,11 +1,29 @@
 import './Home.css';
+import { generateToken } from '@the-collab-lab/shopping-list-utils';
+import { useNavigate } from 'react-router-dom';
+import { useEffect } from 'react';
+
+export function Home({ listToken, setListToken }) {
+	const navigate = useNavigate();
+
+	const createToken = async () => {
+		const token = await generateToken();
+
+		setListToken(token);
```

Comment:
Navigating to the list view only needs to be done once when the "Create new list" button is pressed.

```suggestion
		setListToken(token);
		navigate('/list');
```

Reply:
I'm leaving `navigate` as is in `useEffect` as that handles both cases of the token being newly created and when the token was already created from a previous session because if there is already a token saved in localStorage when the page first loads the user needs to be automatically redirected to the list view.

Reply:
That works. I was under the impression that the user needed to access the home view whenever they wanted to, but looking at the fully built example, it seems that that's not necessary.